charts/sck-otel/values.yaml

```diff
@@ -176,12 +176,15 @@ journaldLogs:
 # keeping the default false until journald receiver is part of splunkotel distro
   enabled: false
   directory: /run/log/journal
-  priority: info
+  defaultPriority: info
+  # List of service units to collect and configuration for each. To collect all journald services logs, leave units blank.
   units:
-    - ssh
-    - kubelet
-    - docker
-    - containerd
+  - name: kubelet
+    priority: info
+  - name: docker
+    priority: info
+  - name: containerd
+    priority: info
```

Comment:
To ingest all the journald service logs, do we leave `units` blank like below? 

> `units: ` 

What are the default values for units?

Reply:
yes. i left a comment to leave it blank to collect all.

Reply:
Are we adding only "kubelet", "docker" and "containerd" units by default? or all?

Reply:
yea, just following what we had in SCK